bullmq-task-queue/package.json

```
@@ -10,19 +10,19 @@
     "@remix-run/node": "*",
     "@remix-run/react": "*",
     "@remix-run/serve": "*",
-    "bullmq": "^1.76.0",
-    "ioredis": "^4.28.5",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2"
+    "bullmq": "^2.1.0",
+    "ioredis": "^5.2.3",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
   },
   "devDependencies": {
     "@remix-run/dev": "*",
     "@remix-run/eslint-config": "*",
-    "@types/ioredis": "^4.28.8",
-    "@types/react": "^17.0.39",
-    "@types/react-dom": "^17.0.13",
-    "eslint": "^8.10.0",
-    "typescript": "^4.7.4"
+    "@types/ioredis": "^4.28.10",
+    "@types/react": "^18.0.21",
+    "@types/react-dom": "^18.0.6",
+    "eslint": "^8.24.0",
+    "typescript": "^4.8.3"
```

Let's keep these the same as well, to be consistent with all other examples

```suggestion
    "eslint": "^8.10.0",
    "typescript": "^4.7.4"
```